js/options.js

```diff
@@ -9,20 +9,21 @@ $(function() {
         }
     });
     $('#save').on('click', function() {
-        getTenants(function(tenants) {
-            for (var tenantCode in tenants) {
-                if (tenantCode === $('#tenantCode').val()) {
-                  saveActiveTenant(tenants[tenantCode], function() {
-                    // Update status to let user know options were saved.
-                    $('#status').html('<div class="alert alert-success">' + chromeOrBrowser().i18n.getMessage('optionsSettingsSaved') + '</div>');
-                    $('#optionsHelp').addClass('hidden');
-                    setTimeout(function() {
-                      $('#status').textContent = '';
-                    }, 750);
-                  });
+        if ($('#specifyTenant:selected').length > 0) {
+            var otherTenantCode = $('#tenantCode').val();
```

Comment:
When the user has selected other and clicks save this block is executed.
I extract the code and region from the input elements and save a tenant that is generated in code.